### PR TITLE
Fix error message display with multiple hyphens.

### PR DESCRIPTION
**Pull request by garg3133**
Fixes: https://github.com/nightwatchjs/nightwatch/issues/3972

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
     "@typescript-eslint/parser": "^5.57.1",
     "@vitejs/plugin-react": "^3.1.0",
     "babel-plugin-styled-components": "^2.0.7",
-    "chromedriver": "^111.0.0",
+    "chromedriver": "^120.0.2",
```
Comment (vaibhavsingh97) [inline package.json:56]: we can remove it, since newer version of Nightwatch doesn't require `chromedriver`

Comment (garg3133) [reply package.json]: Done

File: src/components/SpecMetaData/utils.ts [hunk 2]
```diff
@@ -34,7 +34,12 @@ export const getTestsSteps = (data: IEnvironmentData, fileID: string, testID: st
         status: test.status,
         ...(test.result?.message && { message: test.result.message }),
         ...(test.result?.beautifiedStack && { stacktrace: test.result.beautifiedStack }),
-        ...(test.result?.message && { shortMessage: test.result.message.split('-') }),
+        ...(test.result?.message && {
+          shortMessage: [
+            test.result.message.split(' - ')[0],
+            test.result.message.split(' - ').slice(1).join(' - '),
```
Comment (vaibhavsingh97) [inline src/components/SpecMetaData/utils.ts:40]: is this change needed?

Comment (garg3133) [reply src/components/SpecMetaData/utils.ts:40]: This is the main change in this PR to fix the linked issue. It should only split on the first occurrence of ` - ` and not after that, otherwise the error message gets truncated.

Comment (garg3133) [reply src/components/SpecMetaData/utils.ts:40]: I've checked this change for possible errors and it will never throw an error for any string values of `this.result.message`.

**Review by vaibhavsingh97 — APPROVED** [review]
